src/OrchardCore/OrchardCore.Abstractions/Localization/CultureScope.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Globalization;
+
+namespace OrchardCore.Localization
+{
+    public sealed class CultureScope : IDisposable
+    {
+        private CultureScope(CultureInfo culture)
```

Comment:
Maybe `UICultureScope` as we only set the UI culture, or as suggested by Seb we could set both cultures